### PR TITLE
feat: Trending 2.0

### PR DESCRIPTION
**Pull request by hirbod**
# Why

This PR brings the all new Trending view. 
And its pretty fast :)

# How

# Test Plan

### CODE REVIEW

File: packages/app/components/card/list-card.tsx [hunk 1]
```diff
@@ -0,0 +1,356 @@
...
+
+function ListCard(props: Props) {
+  const { width } = useWindowDimensions();
+  const isLgWidth = width >= breakpoints["md"];
+
+  if (!isLgWidth) {
+    return <ListCardSmallScreen {...props} />;
+  }
+
+  return <ListCardLargeScreen {...props} />;
```
Comment (intergalacticspacehighway) [inline packages/app/components/card/list-card.tsx:99]: not a blocker: i have seen stuff jumping as we use JS based logic for screen resolution mounting. We should probably start using tailwind's `hidden` in future.

File: packages/app/components/card/list-card.tsx [hunk 2]
```diff
@@ -0,0 +1,356 @@
...
+      nft?.token_description
+        ? linkifyDescription(removeTags(nft?.token_description))
+        : "",
+    [nft?.token_description]
+  );
+
+  return (
+    <View
+      // @ts-expect-error TODO: add accessibility types for RNW
+      accessibilityRole="article"
```
Comment (intergalacticspacehighway) [inline packages/app/components/card/list-card.tsx:267]: 💯

File: packages/app/components/card/list-card.tsx (hunk 1 above)
Comment (hirbod) [reply packages/app/components/card/list-card.tsx:99]: This was the logic of the old `Card`, I just optimized it a bit. We can keep optimizing in the future for sure.

**Review by intergalacticspacehighway — APPROVED** [review]
LGTM 🔥🔥🔥

File: packages/app/components/trending/trending.web.tsx [hunk 3]
```diff
@@ -65,56 +71,38 @@ export const Trending = () => {
...
-            new Array(numColumns - chuckItem.length)
-              .fill(0)
-              .map((_, itemIndex) => (
-                <View key={itemIndex.toString()} tw="flex-1" />
-              ))}
+        <View tw="mx-auto max-w-screen-xl flex-row md:px-4">
+          <ListCard
+            nft={item}
+            showClaimButton
+            href={`/${getNFTSlug(
```
Comment (intergalacticspacehighway) [inline packages/app/components/trending/trending.web.tsx]: `getNFTSlug` adds the `slash` so we can remove it from here.

Comment (hirbod) [reply packages/app/components/trending/trending.web.tsx]: fixed

**Pull request merged**